Second draft completed (#5)
arfc/2019-park-math-comp: summary/summary.tex

\begin{document}

\begin{abstract}

> This summary presents the outline for the code validation of Moltres, a coupled neutronics/thermo-hydraulics simulation tool developed for simulating molten salt reactors, using a modified geometry of the Molten Salt Fast Reactor benchmark model. Homogenised neutron group constant data for each material region were generated using the Monte Carlo neutron transport code Serpent. The data will be fed into Moltres, along with other input parameters such as flow velocity and initial conditions. The steady state and selected accident transient simulation results will be studied and compared against existing results for the MSFR benchmark model. With this study, we will assess the performance of the Moltres code, and identify key technological gaps and areas for improvement.

Author: I used "benchmark" because that was the word that Pettersen used to refer to this model. Should I change it to "reference"?

By comparing the simulation results with existing literature on the MSFR, we can quantify and assess the impact of these technological gaps, and develop an effective plan for future work on improving Moltres.

\section{MOLTEN SALT REACTORS}

%A major feature to be introduced is pump and buoyancy driven flow. The current implementation of Moltres requires user-determined velocity values as opposed to having the flow determined by fluid dynamics in the presence of pumps and buoyancy effects due to the expansion of heated molten salt. While this implementation is sufficient for 

> The development of accurate simulation tools for MSRs is an essential step towards their potential deployment. MSRs are arguably much safer than conventional nuclear reactors due to several passive safety mechanisms that make severe accidents extremely unlikely to occur. When core temperatures rise due to unprotected reactivity insertions, the strong negative temperature reactivity coefficients of the fuel salt limit further increases in reactivity and temperature \cite{elsheikh_safety_2013}. The coolant salts are chemically inert and do not react explosively with air or water. Fission products can be continuously removed through online chemical reprocessing and helium sparging. This reduces the radiological impact of a severe accident involving a hull breach and the release of molten salt to the environment \cite{elsheikh_safety_2013}.

Reviewer: ```suggestion
The development of accurate simulation tools for MSRs is an essential step towards their potential deployment. MSR designs commonly incorporate passive safety mechanisms and improved fuel utilization. When core temperatures rise due to unprotected reactivity insertions, the strong negative temperature reactivity coefficients of the fuel salt limit further increases in reactivity and temperature \cite{elsheikh_safety_2013}. The coolant salts are chemically inert and do not react explosively with air or water. Fission products can be continuously removed through online chemical reprocessing and helium sparging. This reduces the radiological impact of a severe accident involving a hull breach and the release of molten salt to the environment \cite{elsheikh_safety_2013}.
```

This paragraph is quite disjointed. I have only edited it to keep it from being over-broad.

> Certain MSR designs also operate with epithermal or fast neutron spectra which can burn off the transuranic elements inevitably produced when using $^{235}$U fuel. Some designs such as the Molten Salt Fast Reactor (MSFR) are designed to breed fissile $^{233}$U from $^{232}$Th \cite{serp_molten_2014}, thereby gaining the advantages of running a thorium fuel cycle. MSRs also typically operate at low pressures compared to light water reactors (LWR) as the operating temperatures are far below the boiling temperatures of the molten salt mixture. This eliminates the need for thick pressure vessels borne from safety constraints \cite{elsheikh_safety_2013}.

Reviewer: ```suggestion
Certain MSR designs also operate with epithermal or fast neutron spectra which can burn off the transuranic elements inevitably produced when using $^{235}$U fuel. Some designs such as the Molten Salt Fast Reactor (MSFR) aim to breed fissile $^{233}$U from $^{232}$Th \cite{serp_molten_2014}, thereby gaining the advantages of running a thorium fuel cycle. MSRs also typically operate at low pressures compared to light water reactors (LWR) as the operating temperatures are far below the boiling temperatures of the molten salt mixture. This eliminates the need for thick pressure vessels borne from safety constraints \cite{elsheikh_safety_2013}.
```

\section{THE MOLTEN SALT FAST REACTOR}

> The MSFR is a reference design for a fast-spectrum molten salt reactor first developed under the Evaluation and Viability of Liquid Fuel Fast Reactor System (EVOL) project, and continued by the Safety Assessment of the Molten Salt Fast Reactor (SAMOFAR) project \cite{serp_molten_2014}. Figure \ref{fig:msfr} shows a schematic view of the MSFR. The main specifications of the MSFR are given in Table \ref{table:msfr}.

Reviewer: ```suggestion
The MSFR reference design was first developed under the Evaluation and Viability of Liquid Fuel Fast Reactor System (EVOL) project, and extended by the Safety Assessment of the Molten Salt Fast Reactor (SAMOFAR) project \cite{serp_molten_2014}. Figure \ref{fig:msfr} shows a schematic view of the MSFR. The main specifications of the MSFR are given in Table \ref{table:msfr}.
```

\begin{table}
\begin{tabular}{ l l }
\hline
Parameter & Value \\
\hline
Electric/Thermal power [MW$_{\text{e}}$/MW$_{\text{th}}$] & 1300 / 3000  \\
Salt volume [m$^3$] & 18 \\
Salt fraction in core & 0.5 \\
Number of circulation loops & 16 \\
Nominal flow rate [kg s$^{-1}$] & 18500  \\
Nominal circulation time [s] & 4.0 \\
Inlet/outlet temperature [K] & 973 / 1073 \\
Blanket volume [m$^3$] & 7.3\\
\hline
\end{tabular}
\caption{Specifications of the MSFR design \cite{serp_molten_2014}.}
\label{table:msfr}
\end{table}

> Lithium fluoride (LiF) is the major component of the fuel and blanket molten salts used for the MSFR. Fissile and fertile isotopes are introduced into the mixture by mole fractions of 77.5\%LiF-22.5\%AcF$_4$, where AcF$_4$ represents actinide fluorides such as uranium and thorium fluorides. The MSFR supports various fuel compositions. It can run on the same $^{235}$U-$^{238}$U fuel used in most conventional LWRs. It can also be run on a mixture of fresh and used uranium fuel containing reprocessed transuranic isotopes \cite{fiorina_investigation_2013}. However, the main configuration of the MSFR is a breeder reactor running on $^{233}$U-$^{232}$Th \cite{merle-lucotte_launching_2011}. Breeding ratios of up to 1.1 have been reported by previous studies on the MSFR \cite{fiorina_molten_2013}. 

Reviewer: ```suggestion
Lithium fluoride (LiF) fuel and blanket molten salts in the MSFR are loaded with fissile and fertile isotopes. The fuel salt contains mole fractions of 77.5\%LiF-22.5\%AcF$_4$, where AcF$_4$ represents actinide fluorides such as uranium and thorium fluorides. The MSFR supports various fuel compositions, including conventional, enriched uranium. It can also be run on a mixture of fresh and used uranium fuel containing reprocessed transuranic isotopes \cite{fiorina_investigation_2013}. However, the primary MSFR design configuration uses a $^{233}$U-$^{232}$Th mixture \cite{merle-lucotte_launching_2011}. With this fuel, the MSFR may reach breeding ratios up to 1.1 \cite{fiorina_molten_2013}. 
```

I'm still a little baffled by your inclusion of this sentence "It can run on the same $^{235}$U-$^{238}$U fuel used in most conventional LWRs." Do you just mean the same fissile isotopes? Note, the enrichment in MSRs is frequently much higher than in LWRs.

> The primary fuel salt flows upwards through the 9 m$^3$ central core region. At the top of the core, the flow separates into 16 smaller external loops, each of which passes through a heat exchanger and is pumped back into the core. The primary heat exchangers transfer heat from the fuel salt to an intermediate salt coolant loop. There are other instrumentation along the external loops for online reprocessing and gas sparging. The core is radially surrounded by a tank of blanket molten salt, with reflectors at the top and bottom of the core. The blanket salt contains fertile isotopes such as $^{232}$Th for fuel breeding. There is a layer of neutron absorbing material behind the blanket tanks to protect the heat exchangers, pumps and other instrumentation from neutron irradiation damage.

Reviewer: ```suggestion
The primary fuel salt flows upward through the 9 m$^3$ central core volume. At the top of the core, the flow separates into 16 smaller external loops, each of which passes through a heat exchanger and is pumped back into the core. The primary heat exchangers transfer heat from the fuel salt to an intermediate salt coolant loop. Additional instrumentation along the external loops enables online reprocessing and gas sparging. A tank of blanket salt radially surrounds the core and reflectors flank it at the top and bottom. The blanket salt contains fertile isotopes such as $^{232}$Th for fuel breeding. A layer of neutron absorbing material behind the blanket tanks protects the heat exchangers, pumps and other instrumentation from neutron irradiation damage.
```

> Various authors have performed a number of steady state and transient multiphysics simulations for the MSFR. A paper by Fiorina et al. \cite{fiorina_modelling_2014} compares results between coupled models developed on the multiphysics code COMSOL and an in-house code developed at Delft University of Technology. The models used 2D axisymmetric models of the MSFR and solved multi-group neutron diffusion equations for the neutronics, and they showed good agreement for some of the transient cases studied. A more comprehensive 3D model has been developed and studied by Aufiero et al. \cite{aufiero_development_2014} using the computional fluid dynamics (CFD) toolbox OpenFOAM. Although this model relied on the one-group neutron diffusion equation, it enabled the study of the full 3D core geometry and 3D transient scenarios such as the failure of one of the 16 pumps in the MSFR.

Reviewer: ```suggestion
Various authors have performed steady state and transient multiphysics simulations of the MSFR.  Fiorina et al. \cite{fiorina_modelling_2014} compare results between coupled models developed with the multiphysics code COMSOL and an in-house code developed at Delft University of Technology. The models used 2D axisymmetric models of the MSFR and solved the multi-group neutron diffusion equations for the neutronics, and they showed good agreement for some of the transient cases studied. A more comprehensive 3D model has been developed and studied by Aufiero et al. \cite{aufiero_development_2014} using the computional fluid dynamics (CFD) toolbox OpenFOAM. Although this model relied on the one-group neutron diffusion equation, it enabled full 3D core geometry and 3D transient scenarios such as the failure of one of the 16 pumps in the MSFR.
```

" they showed good agreement for some of the transient cases studied. "
Good agreement with what?

> More recently, a coupled tool, comprising of the thermal-hydraulics code TRACE and a multi-group 3D spatial neutronics solver PARCS, was used to run steady state and transient simulations of the MSFR \cite{pettersen_coupled_2016}. This approach allowed for a 3D model simulation of the MSFR on a coarse mesh, thus improving on the 2D axisymmetric COMSOL/TUDelft models with the benefit of much lower computation times in comparison to the CFD-OpenFOAM models. On top of steady state simulations, the transient scenarios studied by Pettersen \cite{pettersen_coupled_2016} include loss of heat sink, pump over-speed, over-cooling and loss of flow. 

Reviewer: ```suggestion
More recently, a coupled tool, comprised of the thermal-hydraulics code TRACE and the multi-group 3D spatial neutronics solver PARCS, was used to run steady state and transient simulations of the MSFR \cite{pettersen_coupled_2016}. This approach enabled a 3D MSFR simulation on a coarse mesh, thus improving on the 2D axisymmetric COMSOL/TUDelft models as well as shorter computation times than the CFD-OpenFOAM models. On top of steady state simulations, the transient scenarios studied by Pettersen \cite{pettersen_coupled_2016} include loss of heat sink, pump over-speed, over-cooling and loss of flow. 
```
